mtags/src/main/scala-3/scala/meta/internal/pc/SemanticdbSymbols.scala

```diff
@@ -64,7 +68,7 @@ object SemanticdbSymbols:
                     .alternatives
                     .iterator
                     .map(_.symbol)
-                    .filter(sym => symbolName(sym) == s)
+                    // .filter(sym => symbolName(sym) == s)
```

Comment:
Turns out this line was actually  needed for overloaded methods 😅 
Basically, the problem is: For code like
```scala
// Filename: Main.scala
package a:
  def fooBar(x: Int) = x + 1
  def foobar(x: String) = x.length
```
Semanticdb contains names
```
a/Main$package.fooBar().
a/Main$package.fooBar(+1).
```
But `MtagsIndexer` creates names
```
a/fooBar().
a/fooBar(+1).
```
So I have to fix it in the indexer